google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.storage;
+
+/**
+ * String constants for the names of the storage classes.
+ */
+public class StorageClass {
```

Comment:
Maybe `StorageClasses` instead of `StorageClass`? The name `StorageClass` sounds like it is a storage class, not that it contains them.